mainpage.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
You will need to rename this to `index.html` before you push to your `gh-pages` branch. Otherwise the GitHubz will not cooperate with your deployment.